nfcreader/src/main/java/company/tap/nfcreader/open/reader/TapNfcCardReader.java

```diff
@@ -30,6 +38,12 @@ public TapNfcCardReader(Activity activity) {
         tapNfcUtils = new TapNfcUtils(activity);
         provider = new TapNfcProvider();
         logger = LoggerFactory.getLogger(TapNfcCardReader.class);
+        String app_name = activity.getApplicationInfo().loadLabel(activity.getPackageManager()).toString();
+        // Capture author info & user status
+        Map<String, String> parameters = new HashMap<String, String>();
+        parameters.put("sdkVersion", "1.0");
```

Comment:
We Cann't hardcode the version name, fetch the current actual version name for the module